src/client/views/member.html

```diff
@@ -53,7 +53,7 @@ <h6 class="center">Account Info</h6>
                         </section>
                     </div>
                     <div style="flex-grow: 0;">
-                        <button id="delete-account" class="hidden action-button" onclick="removeAccount()">Delete account</button>
+                        <button id="delete-account" class="hidden action-button" onclick="removeAccount(true)">Delete account</button>
```

Comment:
Instead of doing this, add an event listener on this element in the javascript that listens for the 'onclick' event. This way all javascript stays in our javascript file.